src/App.tsx

```diff
@@ -1,7 +1,68 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+// class Component
+
+type State = {
+  pressedKey: string | null;
+};
+
+export class App extends React.Component {
+  state: State = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
+    });
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
+    });
+  }
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        {pressedKey ? (
+          <p className="App__message">
+            The last pressed key is [
+            {pressedKey}
+            ]
+          </p>
+        ) : (
+          <p className="App__message">
+            Nothing was pressed yet
+          </p>
+        )}
+      </div>
```

Comment:
you can change only a message, since tag and class isn't changed
```suggestion
      <div className="App">
        <p className="App__message">
          {pressedKey ? `The last pressed key is [${pressedKey}]` : 'Nothing was pressed yet'}
        </p>
      </div>
```